internal/handshake/tls_extension_handler_client.go

```diff
@@ -47,6 +57,7 @@ func (h *extensionHandlerClient) Send(hType mint.HandshakeType, el *mint.Extensi
 }
 
 func (h *extensionHandlerClient) Receive(hType mint.HandshakeType, el *mint.ExtensionList) error {
+	utils.Debugf("Received %#v", hType)
```

Comment:
Aside: We should probably note somewhere that DEBUG should not be enabled in prod :)

Reply:
Isn't that the definition of DEBUG? ;)

Regarding that log statement here, I just forgot to remove that.